themes/classic/_dev/css/partials/_helpers.scss

```diff
@@ -0,0 +1,49 @@
+/*** HELPERS ***/
+
+._relative{
+  position: relative;
+}
+
+._dpt{
+  display: table;
+}
```

Comment:
ok, I will change this naming convention and check what bootstrap helpers do the same thing